apps/frontend/core-dapp/src/features/deposit/BalanceBox.tsx

```diff
@@ -58,10 +57,15 @@ const GreenColor = styled.span`
   font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
 `
 
+const PurpleColor = styled.span`
+  color: #6264d9;
```

Comment:
and this should use color from theme, and use light purple in dark mode

```suggestion
  color: ${({theme}): string => theme.color.primaryLight};
```

Reply:
Huh, so I couldn't find this color because on figma it's `#6264d9` and on our codebase it's `#6264d8` 😬